Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -1256,6 +1269,17 @@ namespace gdjs {
       );
     }
   }
+
+  export namespace PlatformerObjectRuntimeBehavior {
```

Comment:
If this possible not to export this State/namespace? To avoid exposing unnecessary details to the outside world.